game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileDescription.java

```diff
@@ -69,7 +69,7 @@ public static DownloadFileDescription ofMapDownloadListing(
         .url(mapDownloadListing.getUrl())
         .description(mapDownloadListing.getDescription())
         .mapName(mapDownloadListing.getMapName())
-        .version(new Version(mapDownloadListing.getVersion()))
+        .version(new Version(mapDownloadListing.getVersion()).getMajor())
```

Comment:
What format does the Version String have here to justify the extra step via the version object?

Reply:
It is a simple number, this could be converted to just be an integer here as well. One ramification though is the `mapDownloadListing#getVersion` would then return a number which then has further changes to code that I think would be deleted soon.

I think we will be able to simplify this soon enough, but holding off for now as upcoming changes could make it an even easier update.